src/storage_manifests.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, charm, charm_config, kube_control, integrator):
                 ConfigRegistry(self),
                 CreateStorageClass(self, "default"),  # creates csi-cinder-default
                 UpdateSecrets(self),  # update secrets
+                UpdateControllerPlugin,
```

Comment:
it's gonna need a reference to the base manifest class
```suggestion
                UpdateControllerPlugin(self),
```

Reply:
thanks!